app/components/ConversationBanner/conversationBanner.js

```diff
@@ -1,41 +1,43 @@
-import React, { Component } from "react";
-import { View, Text, Image, TouchableOpacity } from "react-native";
-import styles from "./style";
+import React, { Component } from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import styles from './style';
```

Comment:
Our standard is to use double quote instead of single quotes. We might need to update the prettier config for this.

Reply:
I have changed the Prettier configuration `.prettierrc.json` 

![Screenshot 2023-03-08 175817](https://user-images.githubusercontent.com/84832565/223713418-813775db-d527-4f0e-a94a-e1fe0aafa582.png)

and run this script `npx prettier --write "**/*.js` to format all the js file.